curriculum/challenges/english/08-coding-interview-prep/rosetta-code/averages-root-mean-square.english.md

```diff
@@ -6,9 +6,10 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<p>Compute the  <a href="https://en.wikipedia.org/wiki/Root mean square" title="wp: Root mean square">Root mean square</a>  of the numbers 1 through 10 inclusive.</p>
-<p>The  root mean square  is also known by its initials RMS (or rms), and as the quadratic mean.</p><p>The RMS is calculated as the mean of the squares of the numbers, square-rooted:</p>
-<p><big>$$x_{\mathrm{rms}} = \sqrt {{{x_1}^2 + {x_2}^2 + \cdots + {x_n}^2} \over n}. $$</big></p>
+Compute the  <a href="https://en.wikipedia.org/wiki/Root mean square" title="wp: Root mean square" target='_blank'>Root mean square</a>  of the numbers 1 through 10 inclusive.
+The <i>root mean square</i> is also known by its initials RMS (or rms), and as the <b>quadratic mean</b>.
+The RMS is calculated as the mean of the squares of the numbers, square-rooted:
+<big>$$x_{\mathrm{rms}} = \sqrt {{{x_1}^2 + {x_2}^2 + \cdots + {x_n}^2} \over n}. $$</big>
 </section>
 
```

Comment:
Same here also, instructions are in descriptions.

Reply:
This problem also had a very short description/instructions, so I left it in the description section.